modules/com_vtiger_workflow/expression_engine/VTExpressionsManager.inc

```diff
@@ -152,6 +152,7 @@ class VTExpressionsManager {
 			'allrelatedare' => 'allrelatedare(relatedmodule, fieldname, value)',
 			'min' => 'min(fieldname)',
 			'max' => 'max(fieldname)',
+			'holidaydifference' => 'holidaydifference(startDate, endDate,mapname, include_saturdays)',
```

Comment:
change this:
- the parameters should be: (startDate, endDate, include_saturdays, mapname)  to be the same as the other holiday functions
- move the whole line to be next to the other holiday functions

Reply:
okay understood